admin/nginx/mbserver-rewrites.conf

```diff
@@ -83,6 +83,7 @@ rewrite ^/list\.html$                            $mb_server/doc/Communication/Ma
 rewrite ^/mod_faq\.html$                         $mb_server/doc/Editing_FAQ                  permanent;
 rewrite ^/mod_intro\.html$                       $mb_server/doc/How_Editing_Works            permanent;
 rewrite ^/news/licenses\.html$                   $mb_server/doc/About/Data_License           permanent;
+rewrite ^/ns/mmd-2.0$                            $mb_server/doc/Development/XML_Web_Service  permanent;
```

Comment:
You need to escape the dot:
`rewrite ^/ns/mmd-2\.0$`

Reply:
Whoops. Done!